Neos.ContentRepositoryRegistry/Classes/Command/NodeMigrationCommandController.php

```diff
@@ -114,6 +114,40 @@ public function kickstartCommand(string $packageKey): void
        $this->outputLine('Your node migration has been created successfully.');
     }
 
+    /**
+     * List available migrations
+     *
+     * @return void
+     * @see neos.contentrepositoryregistry:nodemigration:list
+     */
+    public function listCommand(): void
+    {
+        $this->outputLine();
```

Comment:
We don't typically add an empty line before CLI output, why did you add this?

Reply:
🤷‍♂️ It's copy paste code. I didn't question it. Will remove it.